pyproject.toml

```diff
@@ -92,15 +92,15 @@ colorhash = "~1.0.2"
 pika = "~1.1.0"
 jsonschema = "~3.2"
 packaging = ">=19.0,<20.4"
-gevent = ">=1.4,<1.6"
-pytz = "^2019.1"
+gevent = ">=1.4,<20.6"
```

Comment:
haha, the occasional version jump from 1 to 20 🤣

Reply:
Yes, they simply switched from the normal naming to something like 20.4.0 where 20 is a year and 4 is a month.
https://pypi.org/project/gevent/#history